### PR TITLE
[Improvement #12471] Migrate all dqrule-related interface functions from ProcessServiceImpl

### LINKED ISSUE #12471: [Refactor] Migrate all dqrule-related interface functions from ProcessServiceImpl
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

+ This is a part of #12403 

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by fuchanghai**
## Purpose of the pull request
- this close : #12471 

Migrate all dqrule-related interface functions from ProcessServiceImpl

## Brief change log

Migrate all dqrule-related interface functions from ProcessServiceImpl

## Verify this pull request

This pull request is code cleanup without any test coverage.

### CODE REVIEW

**Comment by DarkAssassinator** [comment]
@fuchanghai need add license head

**Comment by fuchanghai** [comment]
thx for you remind



***@***.***
 
From: Yann Ann
Date: 2022-11-01 22:54
To: apache/dolphinscheduler
CC: fuchanghai; Mention
Subject: Re: [apache/dolphinscheduler] [Improvement #12471] Migrate all dqrule-related interface functions from ProcessServiceImpl (PR #12610)
@fuchanghai need add license head

**Comment by fuchanghai** [comment]
@SbloodyS can you help me to re-run the E2E

**Comment by SbloodyS** [comment]
> @SbloodyS can you help me to re-run the E2E

Done.

**Comment by fuchanghai** [comment]
@DarkAssassinator @ruanwenjun PTAL

File: dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/repository/impl/DqComparisonTypeDaoImpl.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+package org.apache.dolphinscheduler.dao.repository.impl;
+
+import org.apache.dolphinscheduler.dao.entity.DqComparisonType;
+import org.apache.dolphinscheduler.dao.mapper.DqComparisonTypeMapper;
+import org.apache.dolphinscheduler.dao.repository.DqComparisonTypeDao;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
```
Comment (DarkAssassinator) [inline dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/repository/impl/DqComparisonTypeDaoImpl.java]: DAO should use @Repository

File: dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/repository/impl/DqRuleExecuteSqlDaoImpl.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import org.apache.dolphinscheduler.dao.mapper.DqRuleExecuteSqlMapper;
+import org.apache.dolphinscheduler.dao.repository.DqRuleExecuteSqlDao;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class DqRuleExecuteSqlDaoImpl implements DqRuleExecuteSqlDao {
```
Comment (DarkAssassinator) [inline dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/repository/impl/DqRuleExecuteSqlDaoImpl.java]: same as above

File: dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/repository/impl/DqRuleInputEntryDaoImpl.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+import org.apache.dolphinscheduler.dao.repository.DqRuleInputEntryDao;
+import org.apache.dolphinscheduler.dao.utils.DqRuleUtils;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class DqRuleInputEntryDaoImpl implements DqRuleInputEntryDao {
```
Comment (DarkAssassinator) [inline dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/repository/impl/DqRuleInputEntryDaoImpl.java]: same as above

File: dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/repository/impl/DqTaskStatisticsValueDaoImpl.java [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+import org.apache.dolphinscheduler.dao.mapper.DqTaskStatisticsValueMapper;
+import org.apache.dolphinscheduler.dao.repository.DqTaskStatisticsValueDao;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import com.baomidou.mybatisplus.core.conditions.query.QueryWrapper;
+
+@Component
+public class DqTaskStatisticsValueDaoImpl implements DqTaskStatisticsValueDao {
```
Comment (DarkAssassinator) [inline dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/repository/impl/DqTaskStatisticsValueDaoImpl.java]: same as above

**Review by DarkAssassinator — COMMENTED** [review]
LGTM, just need change @Component to @Repository

**Comment by fuchanghai** [comment]
done



---Original---
From: "Yann ***@***.***&gt;
Date: Mon, Nov 21, 2022 19:19 PM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;"State ***@***.***&gt;;
Subject: Re: [apache/dolphinscheduler] [Improvement #12471] Migrate alldqrule-related interface functions from ProcessServiceImpl (PR #12610)




 
@DarkAssassinator commented on this pull request.
 
LGTM, just need change @component to @repository

 
In dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/repository/impl/DqComparisonTypeDaoImpl.java:
 &gt; + * distributed under the License is distributed on an "AS IS" BASIS, + * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. + * See the License for the specific language governing permissions and + * limitations under the License. + */ + +package org.apache.dolphinscheduler.dao.repository.impl; + +import org.apache.dolphinscheduler.dao.entity.DqComparisonType; +import org.apache.dolphinscheduler.dao.mapper.DqComparisonTypeMapper; +import org.apache.dolphinscheduler.dao.repository.DqComparisonTypeDao; + +import org.springframework.beans.factory.annotation.Autowired; +import org.springframework.stereotype.Component; + ***@***.***  
DAO should use @repository
 
 
In dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/repository/impl/DqRuleExecuteSqlDaoImpl.java:
 &gt; ***@***.*** +public class DqRuleExecuteSqlDaoImpl implements DqRuleExecuteSqlDao {  
same as above
 
 
In dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/repository/impl/DqRuleInputEntryDaoImpl.java:
 &gt; ***@***.*** +public class DqRuleInputEntryDaoImpl implements DqRuleInputEntryDao {  
same as above
 
 
In dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/repository/impl/DqTaskStatisticsValueDaoImpl.java:
 &gt; ***@***.*** +public class DqTaskStatisticsValueDaoImpl implements DqTaskStatisticsValueDao {  
same as above

**Pull request closed without merging**